geoh5py/groups/property_group.py

```diff
@@ -137,6 +139,19 @@ def attribute_map(self) -> dict:
         """
         return self._attribute_map
 
+    @property
+    def collect_values(self) -> np.ndarray | None:
+        """
+        The values of the properties in the group.
+        """
+
+        if self._properties is None:
+            return None
+
+        return np.r_[
+            [self._parent.get_data(data)[0].values for data in self._properties]
```

Comment:
I would vote that we keep it as a list here, and let the user decide how to parse the data. Otherwise, we are sort of losing the partition of each child data...

Reply:
We are not losing the partition.
The output shape is (len(children), len(nvertices))
Having a list doesn't change anything, even if we loop over it?

I'm okay passing it as a list anyway.

Reply:
`np.r_` should technically give you a long vector shape (n_child * n_vertices, ). You meant np.c_? Let's do a list anyway, since it will remain a pointer to the values of the each entity. User can decide if they want to duplicate the memory.